docs/requirements.txt

```diff
@@ -0,0 +1,16 @@
+sphinx
+matplotlib>=3.0.0
+numpy==1.19.5
+librosa==0.8.0
+SoundFile==0.10.3.post1
+joblib==0.17.0
+pathlib==1.0.1
+pytsmod==0.3.3
```

Comment:
son los requirements para la vis o para todo?

Reply:
creo que realmente esto no sirve para nada actually

Reply:
let me make sure of that y me lo cargo